src/main/java/fiveavian/proxvc/ProxVCClient.java

```diff
@@ -46,16 +46,16 @@ public class ProxVCClient implements ClientModInitializer {
     private Thread outputThread;
 
     public final KeyBinding keyMute = new KeyBinding("key.mute").bindKeyboard(Keyboard.KEY_M);
-    public final KeyBinding keyPushToTalk = new KeyBinding("key.push_to_talk").bindMouse(4);
-    public final KeyBinding keyVCOptions = new KeyBinding("key.vc_options").bindKeyboard(Keyboard.KEY_V);
-    public final KeyBinding[] keyBindings = {keyMute, keyPushToTalk, keyVCOptions};
+    public final KeyBinding keyPushToTalk = new KeyBinding("key.push_to_talk").bindKeyboard(Keyboard.KEY_V);
+    public final KeyBinding[] keyBindings = {keyMute, keyPushToTalk};
     public FloatOption voiceChatVolume;
     public BooleanOption usePushToTalk;
+    public StringOption audioInput;
```

Comment:
this name still isn't descriptive. `audioInput` would likely mean an input of audio, as in a stream, or a system using audio as input. please change this, recommendations are `selectedSpecifier`, `selectedDevice`, `selectedInputDevice`. please update the translation keys and related ids too.